src/components/ReactTableEditable/presets/0-default.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactTableEditable from '../ReactTableEditable';
+
+export default (
+  <ReactTableEditable
+    uxpId="0"
+    style={{ width: '100%' }}
```

Comment:
That's strange (non-realistic, impractical example) that data is built-in to the component instead of be given in the props in preset

Reply:
Component removed.